stories/core/Tree.stories.tsx

```diff
@@ -0,0 +1,139 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import { Tree, TreeProps, TreeNode, Checkbox } from '../../src/core';
+import { action } from '@storybook/addon-actions';
+import { SvgPlaceholder } from '@itwin/itwinui-icons-react';
+import { NodeData } from '../../src/core/Tree/Tree';
+
+export default {
+  component: Tree,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+  },
+  title: 'Core/Tree',
+} as Meta<TreeProps<TreeData>>;
+
+type TreeData = {
```

Comment:
Maybe rename it to `StoryData` or smth, because at first I thought it is some data type from the Tree. And move it inside the story, so users could see it in code tab.